internal/builders/generic/README.md

```diff
@@ -394,7 +387,7 @@ jobs:
 
       # ========================================================
       #
-      # Step 4: Add a step to generate the provenance subjects
+      # Step 2: Add a step to generate the provenance subjects
```

Comment:
Should this be Step 3?

Reply:
it's step 4, but the previous comment is part of the script above so shows up with a different color. I'll separate steps an code s you suggested in another comment. Thanks!